share/spice/today_in_history/content.handlebars

```diff
@@ -0,0 +1,2 @@
+<div class="year"><div class="title_year h-m text--primary">{{wiki_text_to_html year 0}}</div></div>
+<div id="event" class="string t-m">{{wiki_text_to_html str 1}}</div> 
```

Comment:
"event" isn't very descriptive and could collide with some other thing. `id`s should be unique too